contribute/developer-guide.md

```diff
@@ -33,61 +33,61 @@ We recommend using the Git command-line interface to download the source code fo
 1. Open a terminal and run `git clone https://github.com/grafana/grafana.git`. This command downloads Grafana to a new `grafana` directory in your current directory.
 1. Open the `grafana` directory in your favorite code editor.
 
-For alternative ways of cloning the Grafana repository, please refer to [GitHub's cloning a repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) documentation.
+For alternative ways of cloning the Grafana repository, refer to [GitHub's documentation](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository).
 
-**Warning:** Do not use `go get` to download Grafana. Recent versions of Go have added behavior which isn't compatible with the way the Grafana repository is structured.
+> **Caution:** Do not use `go get` to download Grafana. Recent versions of Go have added behavior which isn't compatible with the way the Grafana repository is structured.
```

Comment:
Shouldn't we use the admonition shortcode for this so it's styled correctly?

Reply:
@jdbaldry I didn't change the syntax of these notes to admonitions because this directory is not a subfolder of docs, so I had no way of running "make docs" to preview the files. Should notes in the "contribute" directory be changed to admonitions?

Reply:
@clayton-cornell As I understand it, the "contribute" directory is not built into static HTML pages, except for the main README. (See how it works at https://grafana.com/docs/grafana/latest/developers/contribute/). Because these pages aren't built, I don't think shortcodes would be processed by Hugo to be transformed into admonitions.

Reply:
Yup, that's correct. We can only use shortcodes in `docs/sources` pages that are rendered as part of the website.

Reply:
Ah... TIL.